shuup_tests/importer/test_product_import.py

```diff
@@ -34,6 +35,12 @@ def test_sample_import_all_match(filename):
     product_type = get_default_product_type()
 
     path = os.path.join(os.path.dirname(__file__), "data", "product", filename)
+    if filename == bom_file:
+        import codecs
+        bytes = min(32, os.path.getsize(path))
+        raw = open(path, 'rb').read(bytes)
+        assert raw.startswith(codecs.BOM_UTF8)
```

Comment:
This is just testing that the test data has the BOM. You should also assert that the importer strips the BOM.